src/cascadia/TerminalControl/TerminalControl.vcxproj.filters

```diff
@@ -1,40 +1,34 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
How did we go from version 15 --> 4? What happened here?

Reply:
Seems that this is an automatic change. I have already changed it back.